site/release-process.html

```diff
@@ -391,9 +391,17 @@ <h4>Remove Old Releases from Development Repository and Mirror Network</h4>
 <pre><code>svn rm https://dist.apache.org/repos/dist/dev/spark/v2.3.1-rc1-bin/ \
   https://dist.apache.org/repos/dist/dev/spark/v2.3.1-rc1-docs/ \
   -m"Removing RC artifacts."
-
 </code></pre>
 
+<p>Make sure to also remove the unpublished staging repositories from the <a href="https://repository.apache.org/">Apache Nexus Repository Manager</a>.</p>
+
+<p>When removing older releases, also update <code>js/download.js</code> to indicate the release is not mirrored
```

Comment:
could we expand on `When removing older releases` happens?
I think given major.minor.patch, the previous patch release is archived when a newer one in the same major.minor is released?
ie, 2.1.3 released, 2.1.2 archived

Reply:
This is already sort of covered by the existing text:

    Spark always keeps two releases in the mirror network: the most recent release on the current and
    previous branches.

That doesn't seem to really correspond to the current mirrored releases (and with the expectation of the Hive tests that were added in 2.2), though, so maybe we should update that policy.

Reply:
right, that doesn't seem to match the ASF mirror behavior now